server/src/main/java/org/elasticsearch/index/query/SearchExecutionContext.java

```diff
@@ -376,16 +370,6 @@ private MappedFieldType fieldType(String name) {
         return fieldType == null ? mappingLookup.getFieldType(name) : fieldType;
     }
 
-    /**
-     *
-     * @param name name of the object
-     * @return can be null e.g. if field is root of a composite runtime field
-     */
-    @Nullable
-    public ObjectMapper getObjectMapper(String name) {
-        return mappingLookup.objectMappers().get(name);
-    }
```

Comment:
This is one method that we've been looking to remove for quite some time isn't it?